Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV POETRY_HOME=/opt/poetry
 ENV PATH="$POETRY_HOME/bin:$PATH"
 ENV PYTHONPATH "${PYTHONPATH}:/app"
 
-RUN apt-get update && apt-get install -y curl
+RUN apt-get update && apt-get install -y curl gdal-bin libgdal-dev gcc g++
```

Comment:
Will this have on impact on a non-apple system, like unused packages or throwing build errors?

Reply:
was only throwing errors on building on apple silicon (new m2 laptop).

but I put the this PR in draft as it still doesn't run.  now the containers are built but it gets stuck on "Creating container". i gotta do some digging to figure that out.  so for right now, I can't run protx-dashboard locally on m2.

Reply:
@taoteg @sophia-massie ,  yeah, i'm not certain. can you confirm that it works on your end?